pegasus/sites.v3/hourofcode.com/i18n/no.yml

```diff
@@ -1,4 +1,4 @@
-'no':
+no:
```

Comment:
we need to discard this change (I'll add a jira item to track figuring out why this keeps happening)